pychromecast/__init__.py

```diff
@@ -349,20 +349,19 @@ def reboot(self):
         """ Reboots the Chromecast. """
         reboot(self.host)
 
-    def volume_up(self):
-        """ Increment volume by 0.1 unless it is already maxed.
+    def volume_up(self, delta=0.1):
+        """ Increment volume by 0.1 (or delta) unless it is already maxed.
         Returns the new volume.
 
         """
-        volume = round(self.status.volume_level, 1)
-        return self.set_volume(volume + 0.1)
+        return self.set_volume(self.status.volume_level + abs(delta))
```

Comment:
I don't like the usage of `abs` as it might be confusing for the user. Let's raise a ValueError instead when delta <= 0